cmd/lotus-chainwatch/processor/mpool.go

```diff
@@ -30,11 +30,10 @@ func (p *Processor) subMpool(ctx context.Context) {
 
 	loop:
 		for {
-			time.Sleep(10 * time.Millisecond)
```

Comment:
@frrist why is this sleep here?

Reply:
IIRC that's my invention. It was here because the we want to process those messages in batches, especially as we get them from mpool one-by-one when we process blocks. The time.After solution is much better